lock-api/src/main/java/com/palantir/lock/client/LeaderElectionReportingTimelockService.java

```diff
@@ -115,6 +120,24 @@ public ConjureStartTransactionsResponse startTransactions(ConjureStartTransactio
                 .logId());
     }
 
+    public LeaderElectionStatistics statistics() {
+        return statisticsWithRegistry(new DefaultTaggedMetricRegistry());
+    }
+
+    @VisibleForTesting
+    LeaderElectionStatistics statisticsWithRegistry(TaggedMetricRegistry metricRegistry) {
+        Snapshot metricsSnapshot = metrics.observedDuration().getSnapshot();
+        LeaderElectionStatistics electionStatistics = LeaderElectionStatistics.builder()
+                .p99(metricsSnapshot.get99thPercentile())
+                .p95(metricsSnapshot.get95thPercentile())
+                .mean(metricsSnapshot.getMean())
+                .count(SafeLong.of(metrics.observedDuration().getCount()))
```

Comment:
There is of course a small race condition here, but it's fine

Reply:
Did some digging and I believe that `metricSnapshot.size()` is equivalent, just without the race condition, so replacing with that.